app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -0,0 +1,10 @@
+<%#
+kind: registration
+name: Global Registration
+model: ProvisioningTemplate
+-%>
+
+if (( $EUID != 0 )); then
```

Comment:
I think it's a bashism and not guaranteed to work on most environments. For example in Ubuntu (which uses dash as a shell) `$EUID` is not set and this is invalid shell:
```console
# podman run --rm -it ubuntu:18.04 /bin/sh
# if (( $EUID != 0 )); then echo bla ; fi   
/bin/sh: 1: !=: not found
```

Reply:
using `id` is probably better for this reason
```sh
if ! [ $(id -u) = 0 ]; then
```

unless someone has better suggestion